### PR TITLE
DevWs for www.easyiot.tech containing containing ZigBee RGBW Bulb

### PR DESCRIPTION
**Pull request by jwg-123**

### CODE REVIEW

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy [hunk 1]
```diff
@@ -103,6 +103,8 @@ metadata {
 		// Ajax Online
 		fingerprint manufacturer: "Ajaxonline", model: "AJ-RGBCCT 5 in 1", deviceJoinName: "Ajax Light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb-2000K-6500K"
 		fingerprint manufacturer: "Ajax online Ltd", model: "AJ_ZB30_GU10", deviceJoinName: "Ajax Light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb-2000K-6500K" // Raw Description: 0B 0104 010D 01 08 0000 0003 0004 0005 0006 0008 0300 1000 00
+        // Shenzhen C-Lux
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: Please fix this indentation (use 2 tabs instead of spaces)

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

**Comment by jwg-123** [comment]
OK



发自 网易邮箱大师




---- 回复的原邮件 ----
| 发件人 | Konrad ***@***.***> |
| 日期 | 2021年11月17日 15:48 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [SmartThingsCommunity/SmartThingsPublic] DevWs for www.easyiot.tech containing containing ZigBee RGBW Bulb (PR #76635) |

@KKlimczukS requested changes on this pull request.

In devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy:

> @@ -103,6 +103,8 @@ metadata {
 		// Ajax Online
 		fingerprint manufacturer: "Ajaxonline", model: "AJ-RGBCCT 5 in 1", deviceJoinName: "Ajax Light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb-2000K-6500K"
 		fingerprint manufacturer: "Ajax online Ltd", model: "AJ_ZB30_GU10", deviceJoinName: "Ajax Light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb-2000K-6500K" // Raw Description: 0B 0104 010D 01 08 0000 0003 0004 0005 0006 0008 0300 1000 00
+        // Shenzhen C-Lux


Please fix this indentation (use 2 tabs instead of spaces)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy [hunk 2]
```diff
@@ -103,6 +103,8 @@ metadata {
 		// Ajax Online
 		fingerprint manufacturer: "Ajaxonline", model: "AJ-RGBCCT 5 in 1", deviceJoinName: "Ajax Light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb-2000K-6500K"
 		fingerprint manufacturer: "Ajax online Ltd", model: "AJ_ZB30_GU10", deviceJoinName: "Ajax Light", mnmn: "SmartThings", vid: "generic-rgbw-color-bulb-2000K-6500K" // Raw Description: 0B 0104 010D 01 08 0000 0003 0004 0005 0006 0008 0300 1000 00
+        	// Shenzhen C-Lux
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: Please use tabs here instead of spaces (as in the line below)

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy (hunk 1 above)
Comment (jwg-123) [reply devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: I have already fixed this indentation,please review , thank you

Comment (KKlimczukS) [reply devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: Yes, but please change spaces to tabs (instead of 4 spaces use 1 tab)

**Review by KKlimczukS — APPROVED** [review]

**Pull request merged**